### PR TITLE
fixing Hub details and adding effective date

### PR DESCRIPTION
**Pull request by BretFisher**
Thanks for this operator!  As I get more info from the Hub team, I'll pass it along to help out.

### CODE REVIEW

**Comment by alexellis** [comment]
Can you explain the layer based rate-limiting?

**Comment by BretFisher** [comment]
Only the manifest is tracked, not layer tarballs. See https://docs.docker.com/docker-hub/download-rate-limit/, which was updated last night. I think blog/faq is also coming next week.

File: README.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ This operator can be used to propagate a single ImagePullSecret to all namespace
 
 ### Why is this operator required?
 
-The primary reason for creating this operator, is to make it easier for users of Kubernetes to consume images from the Docker Hub after [recent pricing and rate-limiting changes](https://www.docker.com/pricing) were brought in, an authenticated account is now required to pull images.
+The primary reason for creating this operator, is to make it easier for users of Kubernetes to consume images from the Docker Hub after [recent pricing and rate-limiting changes](https://www.docker.com/pricing) were announced. Starting November 1st, 2020, an authenticated account will be required to pull beyond these limits:
```
Comment (alexellis) [inline README.md:7]: I can't find anything online re: 1st Nov

This actually says it's live now? https://docs.docker.com/docker-hub/download-rate-limit/

File: README.md [hunk 2]
```diff
@@ -4,10 +4,10 @@ This operator can be used to propagate a single ImagePullSecret to all namespace
...
 
-* Unauthenticated users: 100 layers / 6 hours
-* Authenticated users: 200 layers / 6 hours
+* Unauthenticated users: 100 images / 6 hours
+* Authenticated users: 200 images / 6 hours
```
Comment (alexellis) [inline README.md:10]: I'll update this to pulls

**Comment by alexellis** [comment]
Commit febffed should hopefully cover everything.

**Pull request closed without merging**